conan/tools/cmake/__init__.py

```diff
@@ -1,3 +1,4 @@
 from conan.tools.cmake.toolchain import CMakeToolchain
 from conan.tools.cmake.cmake import CMake
 from conan.tools.cmake.cmakedeps import CMakeDeps
+from conan.tools.cmake.cmakegen import CMakeGen
```

Comment:
Build system? Workflow? Toolset?

Maybe toolchain was a bad name in first place. In cmake it's a kind of bootstrap parameters /initial variables' values to override search rules in enable_language / project.

Maybe InitVars is a better name in Conan instead of toolchain.

Reply:
I agree I kind of like ``Toolchain`` name for the wrapper, but on the other hand ``Toolchain`` name was chosen based on what seems to be the most common usage (CMake like): the toolchain file of a build system is typically a file that defines the compiler, architecture, cross-build parameters, etc. This is basically what the Conan ``settings`` and ``options`` is, so the main responsibility of ``XXXXToolchain`` classes is to map from settings and options to build system specific mechanism. And also, because the file is the one that is passed as ``-DCMAKE_TOOLCHAIN_FILE=conantoolchain.cmake``. In that sense the name seems good for the current usage.